app/pages/lab/subject-sets.jsx

```diff
@@ -65,13 +67,14 @@ SubjectSetsPage.defaultProps = {
 
 SubjectSetsPage.propTypes = {
   createNewSubjectSet: React.PropTypes.func,
-  creationInProgress: React.PropTypes.bool,
+  defaultSubjectSetName: React.PropTypes.string,
   labPath: React.PropTypes.func,
   loading: React.PropTypes.bool,
-  error: React.PropTypes.shape({
+  onPageChange: React.PropTypes.func,
+  subjectSetCreationError: React.PropTypes.shape({
     message: React.PropTypes.string
   }),
-  onPageChange: React.PropTypes.func,
+  subjectSetCreationInProgress: React.PropTypes.bool,
   subjectSets: React.PropTypes.arrayOf(React.PropTypes.object)
 };
```

Comment:
These changes were not mentioned in the PR review, but I used the spread operator to pass the container's state into this component, but then changed the props name slightly (ex: subjectSetCreationInProgress became error). Because of this, I had to change the prop name to match the names of the parent's state.